src/main/scala/is/hail/methods/VEP.scala

```diff
@@ -302,7 +124,14 @@ object VEP {
                       fatal(s"VEP output variant ${ VariantMethods.locusAllelesToString(vepLocus, vepAlleles) } not found in original variants.\nVEP output: $s")
                   }
                 } else {
-                  val a = JSONAnnotationImpex.importAnnotation(JsonMethods.parse(s), vepSignature)
+                  val jv = try {
+                    JsonMethods.parse(s)
+                  } catch {
+                    case _: JsonParseException =>
+                      log.warn(s"vep: failed to parse json: $s")
```

Comment:
FYI I added this to handle some malformed JSON generated by VEP when annotation mouse genomes.